imports/ui/components/edit/edit-collection.jsx

```diff
@@ -106,4 +103,4 @@ let EditCollection = React.createClass({
   }
 })
 
-export default connect(state => state)(EditCollection)
```

Comment:
We're exporting a `connect`ed component, if that's what you mean (apologies if it's not).